package.json

```diff
@@ -24,17 +24,13 @@
 
         "mjmcloug.vscode-elixir", 
 
-        "Orta.vscode-react-native-storybooks", 
-        "Orta.vscode-ios-common-files", 
-
         "jpoissonnier.vscode-styled-components", 
         "mikestead.dotenv", 
         "apollographql.vscode-apollo",
```

Comment:
Should this be kept in?

Reply:
I never needed this, I don't know if anyone is using this in Artsy but didn't see people mentioning it. I will remove it as well

Reply:
probably not, I see that the relay extension already does the same